pkg/handlers/system_intakes.go

```diff
@@ -30,7 +30,13 @@ func (h SystemIntakesHandler) Handle() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		switch r.Method {
 		case "GET":
-			systemIntakes, err := h.FetchSystemIntakes(r.Context())
+			var filterValue models.SystemIntakeStatusFilter
+			statusFilters := r.URL.Query()["status"]
+			if len(statusFilters) == 1 {
+				filterValue = models.SystemIntakeStatusFilter(statusFilters[0])
```

Comment:
If you wanted to make this a bit more user-friendly from the API side, you could upcase the value. So if I queried `system_intakes?status=closed`, this would still filter appropriately.

Reply:
👀